build-tools/docker/check-docker-compose-services.sh

```diff
@@ -0,0 +1,21 @@
+export SAPASSWORD=$(openssl rand -base64 12)
```

Comment:
build-tools is a java project, and looks like this file is more around ci checks. on the deployment-config repo we added a ".ci" folder with scripts like this one. What about doing the same here?

Reply:
Will move.

Reply:
moved to .ci directory

Reply:
Updated the workflow too? It's failing as can't find it.